src/components/organisms/ProjectsTable/ProjectsTable.jsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+// import Button from "../../atoms/Button";
+import ProjectTableRow from "../../molecules/ProjectsTableRow";
+import Projects from "./mockProjects";
+
+const Table = styled.table`
+  width: 100%;
+`;
+
+const ProjectsTable = () => {
+  return (
+    <Table>
+      <tr>
+        <th> Name </th>
+        <th> Description </th>
+        <th> Created At </th>
+      </tr>
+      <tbody>
+        {Projects.map((project) => (
+          <ProjectTableRow project={project} />
+        ))}
```

Comment:
You need to add key prop to avoid console error
```suggestion
        {Projects.map((project, i) => (
          // eslint-disable-next-line react/no-array-index-key
          <ProjectTableRow key={i} project={project} />
        ))}
```
<img width="759" alt="image" src="https://github.com/JustArtur/task_tracker_react_2023/assets/79587920/f6b3b597-f458-4926-bb80-aebb55270b01">